nevergrad/optimization/multiobjective/core.py

```diff
@@ -174,6 +174,16 @@ def pareto_front(
         """
         return self._pf.get_front(size, subset, subset_tentatives)
 
+    def get_min_losses(self) -> tp.List[float]:
+        pf = self._pf.get_raw()
+        min_losses = [np.inf] * len(pf[0].value)
+        for point in pf:
+            point_losses = np.array(point.losses)
+            for i, point_loss in enumerate(point_losses):
+                if point_loss < min_losses[i]:
+                    min_losses[i] = point_loss
```

Comment:
that feels highly inefficient, from my xp with numpy, there's always a nicer way to do it, I woult expect 'np.min([p.losses for p in get_raw()], axis=0))' to work just as well. If not np.min, then np.minimum